lib/ext/adviser.rb

```diff
@@ -0,0 +1,19 @@
+require 'csv'
+
+module AdviserExtension
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def to_csv(_ = {})
+      CSV.generate do |csv|
+        csv << ['Ref. Number', 'Name', 'Firm']
+
+        find_each do |adviser|
+          csv << [adviser.reference_number, adviser.name, adviser.firm.registered_name]
+        end
+      end
+    end
+  end
+end
+
+Adviser.send(:include, AdviserExtension)
```

Comment:
My only opinion was about the use of AdvisersListCsv rather than the module one. The modules are good when it is possible to extend and it is not the case (as I explained above). The problem with just renaming is that than you have a lack of some sort of interface or the duck-typing is lost.